libs/NfcCoreLib/lib/LibNfc/phLibNfc_Hci.c

```diff
@@ -1339,6 +1346,87 @@ static NFCSTATUS phLibNfc_NfceeModeSetProc(void *pContext, NFCSTATUS wStatus, vo
     return wIntStatus;
 }
 
+NFCSTATUS phLibNfc_HciChildDevCommonInitComplete(void* pContext, NFCSTATUS status, void* pInfo)
+{
+    NFCSTATUS wStatus = NFCSTATUS_SUCCESS;
+    UNUSED(pInfo);
+    PH_LOG_LIBNFC_FUNC_ENTRY();
+    if (pContext == NULL)
+    {
+        wStatus = NFCSTATUS_FAILED;
+        PH_LOG_LIBNFC_CRIT_STR("LibNfc context is NULL, %!NFCSTATUS!.", wStatus);
+    }
+    else
+    {
+        pphLibNfc_LibContext_t pLibCtx = (pphLibNfc_Context_t)pContext;
+
+        uint8_t bIndex;
+        wStatus = phLibNfc_SE_GetIndex(pLibCtx, phLibNfc_SeStateInitializing, &bIndex);
+        if (wStatus != NFCSTATUS_SUCCESS)
+        {
+            PH_LOG_LIBNFC_CRIT_STR("No NFCEE in initializing state?!");
+        }
+        else if (status != NFCSTATUS_SUCCESS)
+        {
+            wStatus = status;
+            PH_LOG_LIBNFC_CRIT_STR("NFCEE initialization failed, %!NFCSTATUS!.", wStatus);
+            pLibCtx->tSeInfo.bSeState[bIndex] = phLibNfc_SeStateInvalid;
+        }
+        else if (pLibCtx->tSeInfo.tSeList[bIndex].eSE_Type == phLibNfc_SE_Type_eSE)
```

Comment:
Newer UICCs (I think ETSI v12?) do support APDU pipe right? May be OK for the moment to not support them yet, but I assume that this could would have to change in future to support UICCs that do support APDU pipes?